app/templates/components/event-card.hbs

```diff
@@ -55,10 +55,21 @@
               {{t-var this.tags.[2]}}
             </a>
           {{else}}
-          {{!-- Changes under progress --}}
-            {{#each this.tags as |tag|}}
-              <a href="#">{{t-var tag}}</a>
-            {{/each}}
+            <a href="#"
+              class="link item"
+              {{action this.goToExplore this.event.topic.name}}>
+              {{t-var this.tags.[0]}}
+            </a>
```

Comment:
Why not use href-to helper?

Reply:
oh just skipped my mind. Thanks :+1:

Reply:
@iamareebjamal I encountered same problem as I was getting when I was using LinkTo i.e when clicking on one tag and coming back on the home page again and then click on other tag gets previous tags also by default as if it is saved in cache or something like that so for e.g clicking on ```#Business & Professional``` then back to home and then clicking on ```#baby``` and  then back and clicking on again ```#Business & Professional``` gives baby as filter options as well so to avoid this I have used action instead of LinkTo and same problem is coming with href-to.